languages/ruby/exercises/concept/booleans/ride_pass.rb

```diff
@@ -0,0 +1,15 @@
+class RidePass
+  MINIMUM_HEIGHT = 120
+
+  def issue(height)
+    rand(1000) if height > MINIMUM_HEIGHT
+  end
+
+  def valid?(pass)
+    !pass.nil? && pass != false
```

Comment:
Need to reevaluate this line.  Is this what we want?

Reply:
These method bodies with all be replaced with `raise NotImplementedError`